addons/web_editor/static/lib/odoo-editor/test/spec/editor.test.js

```diff
@@ -963,6 +1047,43 @@ X[]
                         contentAfter: '<p>[]abc</p>',
                     });
                 });
+                it('should ignore ZWS', async () => {
+                    await testEditor(BasicEditor, {
+                        contentBefore: '<p>ab\u200B[]c</p>',
+                        stepFunction: deleteBackward,
+                        contentAfter: '<p>a[]c</p>',
+                    });
+                });
+                it('should keep inline block', async () => {
+                    await testEditor(BasicEditor, {
+                        contentBefore: '<div><p>ab</p><br><span>c[]</span></div>',
+                        stepFunction: deleteBackward,
+                        contentAfter: '<div><p>ab</p><br>[]</div>',
```

Comment:
Would it have kept the span if there was a style on it ? That seems important.

Reply:
yes